spec/models/enterprise_spec.rb

```diff
@@ -145,6 +145,86 @@
       it "sets the enterprise contact to the owner by default" do
         expect(enterprise.contact).to eq enterprise.owner
       end
+
+      context "prevent an wrong instagram link pattern" do
+        it "expects to be invalid the instagram attribute https://facebook.com/user" do
```

Comment:
It's better English to write it like this:
```suggestion
        it "invalidates the instagram attribute https://facebook.com/user" do
```